apps/www/pages/storage/Storage.tsx

```diff
@@ -113,7 +112,11 @@ function StoragePage() {
                 />
               </div>
               <h4 className="h4">Dependable</h4>
-              <p className="p">Enterprise-level scalability and durability.</p>
+
+              <p className="p">
+                Multiple Protocol Support: <br/> S3, TUS (Resumable Uploads), REST<br/>
```

Comment:
```suggestion
                Multiple Protocol Support: <br/> S3, Resumable Uploads and  Standard Uploads<br/>
```

REST is weird cos everything is technically rest and these are the terms we use in our docs currently